### PR TITLE
backend/reqif: export: --multiline-is-xhtml option

### PR DESCRIPTION
**Pull request by stanislaw**

### CODE REVIEW

File: strictdoc/backend/reqif/p01_sdoc/sdoc_to_reqif_converter.py [hunk 1]
```diff
@@ -578,22 +624,42 @@ def _convert_document_grammar_to_spec_types(
         ), "Only one grammar element is currently supported."
 
         for element in grammar.elements:
+            fields_names = element.get_field_titles()
+            statement_field_idx = fields_names.index("STATEMENT")
```
Comment (stanislaw) [inline strictdoc/backend/reqif/p01_sdoc/sdoc_to_reqif_converter.py:628]: This needs to be handled carefully. STATEMENT is a must but could be edge cases.

File: tests/integration/backend/reqif/profiles/p01_sdoc/end_to_end/--multiline-is-xhtml/01_option_is_provided/test.itest [hunk 2]
```diff
@@ -0,0 +1,14 @@
+RUN: %strictdoc export --formats=reqif-sdoc --multiline-is-xhtml %S/sample.sdoc
```
Comment (stanislaw) [inline tests/integration/backend/reqif/profiles/p01_sdoc/end_to_end/--multiline-is-xhtml/01_option_is_provided/test.itest:1]: Implement and write another test where there is `multiline_is_xhtml` in strictdoc.toml.
